bancorprotocol/zapper-studio
# Add carbon defi in apps

## Description
Add Carbon Defi as an app.
It should look like that in the UI
![zapper-carbon-defi](https://github.com/Zapper-fi/studio/assets/8143464/4a78732e-69ac-4be8-a3f0-fd8372b4975e)

@wpoulin could you add this to zapper ? 🙏
Also is it possible to add this description in the app page ? I didn't find where is should be set in the code.
"The Ultimate DEX for Onchain Trading & Liquidity"

## Checklist
- [x] I have followed the [Contributing Guidelines](https://github.com/Zapper-fi/studio/blob/main/CONTRIBUTING.md)

## How to test?
Run
```
pnpm dev
```
Then check : 
http://localhost:5001/apps/carbon-defi/positions?groupIds[]=strategy&network=ethereum
Should display 429 positions, each of them with exactly 2 tokens

http://localhost:5001/apps/carbon-defi/balances?addresses[]=0xb58796c7e02852D46bE2d82c7aCCd4524a43b9dE&networks[]=ethereum
Should have two positions with two tokens inside each
